.circleci/config.yml.in

```diff
@@ -99,6 +99,45 @@ commands:
       - brew_install:
           formulae: libtool
 
+  checkout_and_install:
+    parameters:
+      editable:
+        type: boolean
+        default: true
+    steps:
+      - run:
+          name: Install torch
+          command: >
+            {{ pip_install }}
+            --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+      - checkout
+      - run:
+          name: Install torchvision
+          command: >
+            {{ pip_install }}
+            --no-build-isolation
+            <<# parameters.editable >> --editable <</ parameters.editable >> .
+
+  install_prototype_dependencies:
+    steps:
+      - run:
+          name: Install prototype dependencies
+          command: {{ pip_install }} iopath git+https://github.com/pytorch/data
+
+  run_tests:
```

Comment:
I intentionally didn't touch the unittest workflows for now, because they are a different beast entirely. But you are right, the naming might be confusing. Maybe `run_tests_selective`?

Reply:
OK sounds good. I'd say leave it as is, maybe just add a comment saying that most tests are taken care of differently in the unittests workflows

Reply:
I'll see if we can unify them in a follow-up to this PR.